### DISCUSSION TITLE
Support for nomic-ai/nomic-embed-text-v1.5

### DISCUSSION THREAD

**Discussion by SupreethRao99** [michaelfeil/infinity#128] [Q&A]
Nomic AI latest model nomic-ai/nomic-embed-text-v1.5 requires einops as a dependency as well as requiring sentence transformers >2.4.0. It would be great if Infinity supported the model

**Comment by michaelfeil** [comment]
I'll likely not add `einops` to the docker image, but you are welcome to build you own docker image. Release should be up within 24h .
